src/python/visclaw/plotpages.py

```diff
@@ -62,11 +60,10 @@ def __init__(self):
         self.timeframes_fignos = 'all'
         self.timeframes_fignames = {}
         self.timeframes_prefix = 'frame'
-        
-    
-        self.pageitem_list = []
 
 
```

Comment:
This white-space should be cleaned up but again we should have two lines between function definitions.